lib/fileutils.rb

```diff
@@ -358,25 +358,54 @@ def rmdir(list, parents: nil, noop: nil, verbose: nil)
   module_function :rmdir
 
   #
-  # :call-seq:
-  #   FileUtils.ln(target, link, force: nil, noop: nil, verbose: nil)
-  #   FileUtils.ln(target,  dir, force: nil, noop: nil, verbose: nil)
-  #   FileUtils.ln(targets, dir, force: nil, noop: nil, verbose: nil)
+  # When +src+ is the path to an existing file or directory
+  # and +dest+ is the path to a non-existent file,
+  # creates a hard link at +dest+ pointing to +src+; returns zero:
+  #
+  #   Dir.children('tmp0/')                    # => ["t.txt"]
+  #   Dir.children('tmp1/')                    # => []
+  #   FileUtils.ln('tmp0/t.txt', 'tmp1/t.lnk') # => 0
+  #   Dir.children('tmp1/') # => ["t.lnk"]
+  #
+  # When +src+ is the path to an existing file or directory
+  # and +dest+ is the path to an existing directory,
+  # creates a hard link at <tt>dest/src</tt> pointing to +src+; returns zero:
+  #
+  #   Dir.children('tmp2')               # => ["t.dat"]
+  #   Dir.children('tmp3')               # => []
+  #   FileUtils.ln('tmp2/t.dat', 'tmp3') # => 0
+  #   Dir.children('tmp3')               # => ["t.dat"]
+  #
+  # When +src+ is an array of paths to existing files or directories
```

Comment:
Just like above, I don't think `src` can contain paths of directories (only files).

Reply:
Fixed.  (Actually there were three of these, all now fixed; hope that's correct.)